### ISSUE TITLE
Errors from 'Object' folder

### ISSUE THREAD

**Issue by johndrosa** [dotnet/core#6928]
.NET: v6.0.1
VS Code: v1.62.2
Extensions: C# v1.23.16 | .NET install Tool for Extension Authors v1.4.0 | NuGet Package Manager GUI for VSCode v1.1.9

I ran `dotnet new console -o newApp` via command prompt
From VSCode I open a terminal and run `dotnet run`
The code runs as expected, but VSCode also shows 22 Problems and I cannot find any information to clear them

![image](https://user-images.githubusercontent.com/38889348/142095519-2c022663-2075-4fc1-a4a6-a415fa17ed1e.png)

![image](https://user-images.githubusercontent.com/38889348/142095666-9a4aef43-759d-4a70-bb17-a5895016fac2.png)

**Comment by johndrosa** [comment]
I was provided the following document [https://github.com/dotnet/cli-lab/releases](url) to download the `.msi` tool for uninstalling .NET sdk and runtimes.
I ran `dotnet-core-uninstall remove --all-but-latest --sdk` and removed the following sdks

![image](https://user-images.githubusercontent.com/38889348/142131728-d93ddcf5-3dc0-4da2-b3f1-a4b0ccb0cdb4.png)

Once those were removed, I restarted VSCode and now have 0 errors.

![image](https://user-images.githubusercontent.com/38889348/142131832-34369b9c-4461-408c-99c5-10ae4b06fb35.png)

**Comment by mairaw** [comment]
@dsplaisted is there anything with the SDKs that could be causing these errors? The only thing I could think here is that there was a global.json on the directory tree somewhere setting the version to a lower version given the `dotnet --info` command was not ran on the project directory on the screenshot above.

**Comment by dsplaisted** [comment]
Yes, it sounds like it might be something with global.json.  Not sure though.

@DustinCampbell to route as appropriate.

**Comment by ElectricHavoc** [comment]
I was helping him troubleshoot this. We checked dotnet --info, everything pointed to 6. The command line was totally fine. If there was any global json, it could have been only in something omnisharp picked up. We tried installing vscode-insiders for a clean start (same issue). Tried a lot of things, then recommended he clean up because I saw an older sdk from the --info output.

If I were a betting man, I wouldn't have thought a sdk cleanup would have fixed that issue. (Nothing else changed, so I don't know how a global.json would have caused that) - and the cross contamination to fresh insiders build was also sus.

**Comment by johndrosa** [comment]
After rebooting my pc, I didn't work on this issue yesterday. When I fired up VSCode today the errors resurfaced. 
I tested everything in Windows Sandbox and it all worked without fail.
I uninstalled .NET, removed the C# extension, deleted the files I was working with previously.

Downloaded .NET again, created new folders, and ran
`dotnet new console -o newAPP`

Next, I ran 
`cd newApp`

Then
`code .`

From VSCode, I opened a terminal and ran
`dotnet run`

Everything ran as expected and no errors appeared, but I never got the prompt to install the C# extension.
I went in and installed the C# extension and the errors reappeared. 
I removed the extension again and the errors disappeared.

**Comment by DustinCampbell** [comment]
Adding @JoeRobich for the C# for VS Code issue.

**Comment by JoeRobich** [comment]
@johndrosa Can you please share the OmniSharp log from the Output pane from the environment that reproduces this issue?

**Comment by johndrosa** [comment]
![image](https://user-images.githubusercontent.com/38889348/142550651-59f37a8c-4738-49a7-b649-1b7e375f309a.png)

**Comment by JoeRobich** [comment]
OmniSharp is using your install Visual Studio 2019 which does not support .NET 6 SDK. As a workaround you tell O# to only use the packaged MSBuild by adding an "omnisharp.json" file either in your project folder or, if you want it to apply to all projects, to your windows User folder that contains the following:
```json
{
    "msbuild": {
        "UseBundledOnly": true
    }
}
```

**Comment by ElectricHavoc** [comment]
That's so interesting it would try VS by default, even on a fresh install, especially on a path that is using .NET6 (or any other version). You would think it should be the opposite. Really good to know. And knowing is half the battle. 📚

**Comment by johndrosa** [comment]
@JoeRobich I was already in the process of removing VS 2019 when I got your message. Here is the updated log and the error has cleared.

![image](https://user-images.githubusercontent.com/38889348/142554051-bfd3e83b-c149-4de7-8bf7-6b9eb1b7e6cc.png)
